src/require.js

```diff
@@ -3,6 +3,7 @@ require('logging');
 require('config_brain_memory');
 require('config_brain_nextroom');
 require('config_brain_squadmanager');
+require('config_local');
```

Comment:
This needs to be encapsulated in a `try{ } catch (e) {}` block, if no `config_local.js` exit.